api/user.go

```diff
@@ -64,23 +61,15 @@ func (a *API) UserList(w http.ResponseWriter, r *http.Request) error {
 		return badRequestError("Bad Pagination Parameters: %v", err)
 	}
 
-	rows, err := query.Offset(offset).Limit(limit).Find(&users).Rows()
-	if err != nil {
+	query = query.Select("" +
```

Comment:
personally I prefer the style like:
```
queryStr := fmt.Sprintf("COUNT(%s)......", orderTable, orderTable, userTable)
if err := query.Select(queryStr).Offset(offset).Limit(limit).Find(&users).Error; err != nil {
...
}
```

Similar as to above. But it is fine in this case as it is with the style (😭) of the project.

Reply:
You are right, that would have been way better. Just didn't think of it because it was already done with the string concat. Will keep it in mind!
Seemed like nobody wants to be to conservative about an existing style. We should try to improve it over time, right?